doc/release-notes-1.8.rst

```diff
@@ -95,20 +190,27 @@ JIRA Related Items
 
 .. comment  List derived from Jira query: ...
 
+Zephyr Kernel 1.8.0
+The following sections provide a detailed list of changes, by component, since kernel version 1.7.0.
+
 * :jira:`ZEP-248` - Add a BOARD/SOC porting guide
 * :jira:`ZEP-339` - Tickless Kernel
+* :jira:`ZEP-472` - Ethernet packets are getting missed if sent in quick succession.
 * :jira:`ZEP-540` - add APIs for asynchronous transfer callbacks
+* :jira:`ZEP-616` - OS X setup instructions not working on El Capitan
 * :jira:`ZEP-628` - Validate RPL Routing node support
+* :jira:`ZEP-634` - document IRQ handling internals
 * :jira:`ZEP-638` - feature to consider: flag missing functionality at build time when possible
 * :jira:`ZEP-720` - Add MAX30101 heart rate sensor driver
 * :jira:`ZEP-828` - IPv6 - Multicast Join/Leave Support
 * :jira:`ZEP-843` - Unified assert/unrecoverable error infrastructure
 * :jira:`ZEP-888` - 802.15.4 - Security support
 * :jira:`ZEP-932` - Adapt kernel sample & test projects
 * :jira:`ZEP-948` - Revisit the timeslicing algorithm
-* :jira:`ZEP-973` - Remove deprecated API related to device PM, DEVICE\_ and * SYS\_* macros
+* :jira:`ZEP-973` - Remove deprecated API related to device PM, DEVICE_ and * SYS_* macros
```

Comment:
DEVICE\_ and * SYS\_*  (need to escape the _ to keep Sphinx from thinking these are label reference links)